cypress/support/page_objects/part2/insertAddressPage.js

```diff
@@ -0,0 +1,91 @@
+class insertAddressPage_ {
+    continueButton = 'button[title="Continue"][type="submit"]';
+    dropdownWithError = 'div[class="form-group has-error"]';
+    firstNameInput = 'input#AddressFrm_firstname';
+    lastNameInput = 'input#AddressFrm_lastname';
+    addressOneInput = 'input#AddressFrm_address_1';
+    cityInput = 'input#AddressFrm_city';
+    regionStateDropdown = 'select#AddressFrm_zone_id';
+    postCodeInput = 'input#AddressFrm_postcode';
+    mandatoryFields = [this.firstNameInput, this.lastNameInput, this.addressOneInput, this.cityInput, this.postCodeInput]
+
+    generateString(length) {
+        let result           = '';
+        const characters       = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+        const charactersLength = characters.length;
+        for ( let i = 0; i < length; i++ ) {
+            result += characters.charAt(Math.floor(Math.random() *
+                charactersLength));
+        }
+        return result;
+    }
```

Comment:
masz to samo w comandach, zdublowany kod

Reply:
Wiem, bo to w commandach mi nie działało jak próbowałem stamtąd importować, dlatego dałem bezpośrednio do klasy. Jest nawet komentarz na 28 linii commands żeby spytać czy wiesz czemu nie działa ;)

Reply:
A poza tym to usunę, tak